project/player_character/player_character.gd

```diff
@@ -15,7 +15,7 @@ onready var summoning_power := $SummoningPower as GhostSummonerComponent
 onready var door_exit = Vector2.ZERO
 onready var can_travel = false
 onready var available_action = PlayerActions.NONE setget action_setter
-onready var current_room
+onready var current_room_info = SpookyRoomInfo
```

Comment:
I guess this is currently unused? But I wonder if the player will want the position, or if they need a handle on the kind of room they're in.

Reply:
Knowing the room you're in might be useful for like, checking if you're in the room with a ghost if that's how that works. But we could do that with this I guess, as I continue thinking. With like an enum for room name or something,